ITEM.tsv

```diff
@@ -12,7 +12,7 @@ ITEM_20150317_000011
 ITEM_20150317_000012	One-Handed Sword [Slash]
 ITEM_20150317_000013	A weapon that is commonly used and carried by soldiers of the kingdom.
 ITEM_20150317_000014	Gladius
-ITEM_20150317_000015	This weapon is crafted with room for improvement over its military version.
+ITEM_20150317_000015	This crafted weapon is inferior to the version used by the military.
```

Comment:
The crafted weapon is not inferior. The weapon was crafted with the ability to be improved upon and could/could not be inferior. Directly saying its inferior is nowhere intended in the original translation. The original translation is fine as it is.